packages/web-config-server/src/apiV1/utils/findAccessibleGroupedMapOverlays.js

```diff
@@ -54,31 +58,66 @@ const findNestedGroupedMapOverlays = async (
       const mapOverlayGroupRelation = mapOverlayGroupRelations[i];
       const name = mapOverlayGroupIdToName[mapOverlayGroupRelation.child_id];
       const childMapOverlayGroupRelations = await mapOverlayGroupRelation.findChildRelations();
-      const children = await findNestedGroupedMapOverlays(
+      const mapOverlays = await findNestedGroupedMapOverlays(
```

Comment:
We do support nested map overlay groups (in laos schools: https://github.com/beyondessential/tupaia-backlog/issues/1788#issuecomment-784827175). So the original `children` here actually can have map overlay groups as well. So I think naming this `mapOverlays` may not be accurate